src/main/java/seedu/duke/Parser.java

```diff
@@ -110,24 +111,27 @@ public static Command checkAction(String inputCommand) throws NumberFormatExcept
             if (isProjectListView) {
                 System.out.println("Not in Task View!"); //----------REPLACE WITH EXCEPTION
             } else {
-                for (int i = 1; i < inputs.length - 1; i++) { //Task name after task keyword and before date
+                for (int i = 1; i < inputs.length; i++) { //Task name after task keyword and before date
                     description += inputs[i];
```

Comment:
You can check my implementation for this variable so that when we print out the description, there will be spacing in between the words.

Reply:
Yup done!